app.py

```diff
@@ -31,18 +31,18 @@ def signup():
 
     if form.validate_on_submit():
         
-        first_name=form.first_name.data,
```

Comment:
The commas behind this will return a tuple
example
```
first_name = "candy",
print(first_name) # ("candy",)
```
we only want a string here